util/regexp-assemble/data/932175.data

```diff
@@ -0,0 +1,27 @@
+##! Please refer to the documentation at
+##! https://coreruleset.org/docs/development/regexp_assemble/.
+
+##! Prevent command aliasing
+
+##! starting tokens prefix
+##!> assemble
+\b
+##!=>
+    ##!> cmdline unix
+alias
+    ##!<
+##!=>
+
+##! match possible white space between prefix expressions
+\b\s+
+##!=>
+
+##! match the alias name
+[\w!%,@]+\b
```

Comment:
What about quotes? The following is valid: `alias "foo"="ls"`.

Reply:
Added single and double quotes.